### PR TITLE
Fixup the QSPI/Hyperflash nodes on i.mx RT boards

### PR DESCRIPTION
**Pull request by galak**
The flash nodes were not correct for the cases of qspi or hyperflash.

Fixup the DTS nodes and flash generation script to handle things properly.

### CODE REVIEW

File: boards/arm/mimxrt1020_evk/mimxrt1020_evk.dts [hunk 1]
```diff
@@ -58,9 +58,11 @@
 };
 
 &flexspi0 {
+	reg = <0x402a8000 0x4000>, <0x60000000 0x800000>;
 	qspi0: qspi@60000000 {
```
Comment (MaureenHelm) [inline boards/arm/mimxrt1020_evk/mimxrt1020_evk.dts]: as a proper spi child, should `qspi@60000000` use the chip select instead of the memory mapped address?

File: boards/arm/mimxrt1020_evk/mimxrt1020_evk.dts [hunk 2]
```diff
@@ -58,9 +58,11 @@
...
-		/* ISSI IS25LP064A-JBLE */
-		reg = <0x60000000 0x800000>;
+		/* ISSI IS25WP064AJBLE */
```
Comment (MaureenHelm) [inline boards/arm/mimxrt1020_evk/mimxrt1020_evk.dts]: This comment can now be removed. The compatible makes it clear

File: boards/arm/mimxrt1050_evk/mimxrt1050_evk_qspi.dts [hunk 3]
```diff
@@ -11,17 +11,19 @@
 #if defined(CONFIG_CODE_ITCM)
 		zephyr,flash = &itcm0;
 #elif defined(CONFIG_CODE_QSPI)
-		zephyr,flash = &qspi0;
+		zephyr,flash = &is25wp064;
```
Comment (MaureenHelm) [inline boards/arm/mimxrt1050_evk/mimxrt1050_evk_qspi.dts]: Why did you change this for only one board?

Comment (galak) [reply boards/arm/mimxrt1050_evk/mimxrt1050_evk_qspi.dts]: Mistake, I will fix.

File: boards/arm/mimxrt1020_evk/mimxrt1020_evk.dts (hunk 2 above)
Comment (galak) [reply boards/arm/mimxrt1020_evk/mimxrt1020_evk.dts]: Will remove.

**Comment by galak** [comment]
@MaureenHelm changes made, can you re-review.

**Review by MaureenHelm — APPROVED** [review]
